src/components/HappHosting/HappHosting.connector.js

```diff
@@ -24,20 +23,31 @@ const hostingUser = graphql(HostingUserQuery, {
   props: ({ data: { hostingUser } }) => ({ hostingUser })
 })
 
-const registerHostingUser = graphql( RegisterHostingUserMutation, {
+const registerHostingUser = graphql(RegisterHostingUserMutation, {
   props: ({ mutate }) => {
     return {
-      // NOTE: Currently host_doc is not validated and the content is irrelevant to the dna...
-      registerHostingUser: ({host_doc}) => mutate({
+      // NOTE: Currently hostDoc is not validated and the content is irrelevant to the dna...
```

Comment:
renamed this variable from `host_doc` to `hostDoc` everywhere as Standard JS wants everything to be camel case.